src/app/app.component.html

```diff
@@ -1,2 +1,6 @@
 <p>Hola bienvenidos</p>
-<app-keyboard></app-keyboard>
\ No newline at end of file
+<<<<<<< HEAD
+<app-result-box></app-result-box>
+=======
+<app-keyboard></app-keyboard>
+>>>>>>> dev
```

Comment:
Esto se te ha colado, al resolver los conflictos
```suggestion
<app-result-box></app-result-box>
<app-keyboard></app-keyboard>
```